pyFAI/gui/calibration/ExperimentTask.py

```diff
@@ -334,22 +334,15 @@ def createImageDialog(self, title, forMask=False, previousFile=None):
         dialog.setWindowTitle(title)
         dialog.setModal(True)
 
-        extensions = collections.OrderedDict()
-        extensions["EDF image files"] = "*.edf"
-        extensions["TIFF image files"] = "*.tif *.tiff"
-        extensions["NumPy binary files"] = "*.npy"
-        extensions["CBF files"] = "*.cbf"
-        extensions["MarCCD image files"] = "*.mccd"
+        builder = FilterBuilder.FilterBuilder()
+        builder.addImageFormat("EDF image files", "edf")
+        builder.addImageFormat("TIFF image files", "tif tiff")
+        builder.addImageFormat("NumPy binary files", "npy")
```

Comment:
npz is also used by numpy...
```suggestion
        builder.addImageFormat("NumPy binary files", "npy npz")
```